Contributing/How-to-Assist-with-Comms.md

```diff
@@ -1,9 +1,10 @@
-## Contribute to Communication and Outreach:  
+## Contribute to Communication and Outreach:
 
-1. A project cannot communicate enough. Reach out to new users or existing collaborators is crucial.
-1. If you have experience and want to help, please send an email to info@moja.global 
-1. Use a clear and descriptive title
-1. Please provide background or references to prove your good intentions if you do not have a track record on GitHub
-1. Attach files, screenshots and animated GIFs to illustrate your ideas
-1. Please allow a few days for *moja global* to reply
- 
+moja global supports new contributors and people willing to help simplify our processes and workflow. If you would like to help us with communication and outreach tasks, please have a look at the following points to get started:
+
```

Comment:
Add something related to social media like LinkedIn or Twitter. Non-coders won't be using GitHub. 
Discourse + social media may work good together.

Reply:
Discourse is not set up as of now. I will add LinkedIn and Twitter right now here.

Reply:
Can you please add links to Slack as well, particularly for the admin queries above.

Reply:
Fixed!